scripts/branch_cleanup.py

```
@@ -0,0 +1,109 @@
+        return {pr["headRefName"] for pr in prs}
+    return set()
+
+
+def get_branch_last_commit(branch):
+    """Get the last commit date for a branch"""
+    cmd = f"git log -1 --format=%cd --date=format:'%Y-%m-%d %H:%M:%S' origin/{branch} 2>/dev/null"
+    output = run_command(cmd)
+    if output:
+        try:
+            return datetime.strptime(output, "%Y-%m-%d %H:%M:%S")
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)